extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisorTest.java

```diff
@@ -3668,6 +3840,59 @@ public TestableKafkaSupervisor(
           spec,
           rowIngestionMetersFactory
       );
+      this.consumerProperties = spec.getIoConfig().getConsumerProperties();
+    }
+
+    @Override
+    protected RecordSupplier<Integer, Long> setupRecordSupplier()
+    {
+      final Map<String, Object> consumerConfigs = new HashMap<>();
```

Comment:
The code here can be simplified by initializing `consumerConfigs` to `KafkaConsumerConfigs.getConsumerProperties()` and then just overriding `metadata.max.age.ms`.